fdbclient/include/fdbclient/TenantSpecialKeys.actor.h

```diff
@@ -139,7 +141,13 @@ class TenantRangeImpl : public SpecialKeyRangeRWImpl {
 
 		std::vector<Future<Void>> createFutures;
 		for (auto const& [tenant, config] : tenants) {
-			createFutures.push_back(createTenant(ryw, tenant, config, nextId++, tenantGroupNetTenantDelta));
+			createFutures.push_back(
+			    createTenant(ryw,
+			                 tenant,
+			                 config,
+			                 nextId++,
+			                 tenantGroupNetTenantDelta,
+			                 ryw->getTransactionState()->cx->clientInfo->get().isEncryptionEnabled));
```

Comment:
We are already passing in `ryw`, so you could possibly make this a little simpler by omitting this last argument and reading it in `createTenant` instead.